ods_ci/utils/scripts/util.py

```diff
@@ -81,13 +81,11 @@ def execute_command(cmd: str, print_stdout: bool = True) -> str | None:
     return None
 
 
-def oc_login(ocp_console_url, username, password, timeout=600):
+def oc_login(ocp_api_url, username, password, timeout=600):
     """
     Login to test cluster using oc cli command
     """
-    cluster_api_url = ocp_console_url.replace("console-openshift-console.apps", "api")
-    cluster_api_url = re.sub(r"/$", "", cluster_api_url) + ":6443"
-    cmd = "oc login -u {} -p {} {} --insecure-skip-tls-verify=true".format(username, password, cluster_api_url)
+    cmd = "oc login -u {} -p {} {} --insecure-skip-tls-verify=true".format(username, password, ocp_api_url)
```

Comment:
I think it could be more readable like:
```suggestion
    cmd = f"oc login -u {username} -p {password} {ocp_api_url} --insecure-skip-tls-verify=true"
```

Reply:
done